lib/fathead/react_native/trigger_words.txt

```diff
@@ -0,0 +1,2 @@
+react mobile
+react-native
```

Comment:
I spoke with @moollaza. If you just put `react-native` and `react native` in for now. We're not entirely sure what the case is with trigger words. 👍